setup.py

```diff
@@ -68,6 +68,7 @@ def find_files(pattern, path=None, root="rez"):
     packages=find_packages('src', exclude=["build_utils",
                                            "build_utils.*",
                                            "tests"]),
+    install_requires=['setuptools'],
```

Comment:
With this new PR, I think this is my only issue. I'll take the branch, erase setuptools on my local host, and see if I'm still able to install rez. If I can't, I do think we need to call it out on the Installation page. It's not exactly an onerous requirement, but a requirement nonetheless.

Reply:
Interesting - I erased setuptools on my local box, and it still installed... How the heck?

Reply:
>Interesting - I erased setuptools on my local box, and it still installed... How the heck?

@maxnbk `install_requires` means "dependencies at runtime", not "dependencies that are needed to install the project".

But now I'm curious, why do we need setuptools at runtime?

Reply:
To be brutally honest i am not entirely sure we do, but i think we do. Here is an example of a failed run:

https://github.com/instinct-vfx/rez/actions/runs/14479919556/job/40614454516#step:6:35

Note that it failed on the "install rez with rez-pip" step not on the actual install. Does that not mean we need setuptools in rez's venv for good?